File: JS/PreenchimentoDeVetor2.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const OBJ = require('./PreenchimentoDeVetor2.js')

TEST('Starting PreenchimentoDeVetor2 test...', (t) => {
    //Test 1
    OBJ.printAllVector2(3)

    t.end()
})

File: GO/PreenchimentoDeVetor2_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import "testing"

func TestPreenchimentoDeVetor2(t *testing.T) {
	tables := []struct {
		X int
	}{
		{2},
		{20},
	}

	for _, table := range tables {
		printAllVector2(table.X)
	}
}
